public/docs/ts/latest/guide/index.jade

```diff
@@ -1,10 +1,501 @@
+  1. [Displaying Data](displaying-data.html)
+  1. [User Input](user-input.html)
+  1. [Forms](forms.html)
+  1. [Dependency Injection](dependency-injection.html)
+
+  We might pause at that point to absorb what we've learned and perhaps experiment on our own before diving into [Template Syntax](template-syntax.html). 
+  That chapter is rather dense. It's also essential reading at some point and
+  we will return to it frequently as we compose our HTML templates, both for understanding and as a reference.
+
+  Follow your own star from there, wherever it leads.
+  
+.l-main-section
+<a id="architecture"></a>
+:marked
+  # Architecture Overview
+  Angular 2 is a framework to help us build client applications in HTML and JavaScript.
+  
+  The framework consists of several cooperating libraries, some of them core and some optional.
+  
+  We write applications by composing HTML templates with Angularized-markup, 
+  writing "Component" classes to manage those templates, adding application logic in services,
+  and handing the top root component to Angular's bootstrapper. 
+  
+  Angular takes over, presenting our application content in a browser and responding to user interactions
+  according to the instructions we provided.
+  
+figure
+  img(src="/resources/images/devguide/intro/airplane.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
+:marked
+  Of course there is more to it than this. 
+  We're cruising at high altitude in this chapter. 
+  We're looking for landmarks. We should expect the details to be fuzzy and obscured by occasional clouds.
+
+  An Angular 2 application rests on seven main building blocks:
+  1. [Component](#component)
+  1. [Template](#template)
+  1. [Metadata](#metadata)
+  1. [Data Binding](#data-binding)
+  1. [Directive](#directive)
+  1. [Service](#service)
+  1. [Dependency Injection](#dependency-injection)
+ 
+figure
+  img(src="/resources/images/devguide/intro/overview.png" alt="overview" style="margin-left:-40px;" )
+:marked
+  Learn these seven and we're on our way.
+
+.l-main-section
+<a id="component"></a>
+:marked
+  ## The Component
+figure
+  img(src="/resources/images/devguide/intro/hero-component.png" alt="Component" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
+:marked
+  A **Component** controls a patch of screen real estate that we could call a "view". 
+  The shell at the application root with navigation links, that list of heroes, the hero editor ... 
+  they're all views controlled by Components.
+  
+  We define a Component's "application logic" - what it does to support the view - inside a class.
+  The class interacts with the view through an API of properties and methods. 
+  
+  <a id="component-code"></a>
+  A `HeroListComponent`, for example, might have a `heroes` property that returns an array of heroes
```

Comment:
It's writer's choice: singular or plural.  We're allowed to say "HeroList" to describe a list of heroes; the "list" part conveys the plurality.  "HeroesList" is equally correct.  I'm keeping it ;-)